x-pack/test/upgrade/apps/dashboard/dashboard_smoke_tests.ts

```diff
@@ -41,6 +42,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         dashboardTests.forEach(({ name, numPanels }) => {
           it('should launch sample ' + name + ' data set dashboard', async () => {
             await PageObjects.home.launchSampleDashboard(name);
+            await PageObjects.timePicker.setCommonlyUsedTime('Last_1 year');
             await PageObjects.header.waitUntilLoadingHasFinished();
             await renderable.waitForRender();
             const todayYearMonthDay = moment().format('MMM D, YYYY');
```

Comment:
Can you also remove the lines that set the absolute time so we just leave the `setCommonlyUsedTime('Last_1 year')`